nuxeo-drive-client/nxdrive/data/ui5/js/ndrive-conflicts.js

```diff
@@ -54,16 +54,27 @@ ConflictsController.prototype.updateErrors = function($scope) {
 	$scope.ignoreds = ignoreds;
 }
 ConflictsController.prototype.updateConflicts = function($scope, $interval) {
+    var lastContributorPending = false;
 	self = this;
 	$scope.conflicts = angular.fromJson(drive.get_conflicts());
 	for (var i=0; i<$scope.conflicts.length; i++) {
+		lastContributorPending = lastContributorPending || $scope.conflicts[i].need_refresh;
 		if ($scope.conflicts[i].last_error == "DUPLICATING") {
 			setTimeout( function() {
 				self.updateConflicts($scope, $interval);
 				$scope.$apply();
 			}, 1000);
 		}
 	}
+	if (lastContributorPending) {
```

Comment:
Based on comments below, I don't think you need to delay for a retry here, just use whatever is already in the db.